src/views/transaction/components/transaction-form/transaction-form.view.jsx

```diff
@@ -497,7 +497,7 @@ function TransactionForm ({
                       <div className={classes.amountButtonsItem}>
                         <p>
                           <span>{showInFiat ? (getFixedTokenAmount(amount || 0, account.token.decimals)) : (amountFiat || 0).toFixed(2)} </span>
-                          <span>{(showInFiat) ? account.token.symbol : preferredCurrency}</span>
+                          <span>{showInFiat ? account.token.symbol : preferredCurrency}</span>
```

Comment:
Maybe this would be more readable rewriting it like this?

```js
{
  showInFiat
    ?
      <p>
        <span>{getFixedTokenAmount(amount || 0, account.token.decimals)}</span>
        <span>{account.token.symbol}</span>
      </p>
    :
      <p>
        <span>{(amountFiat || 0).toFixed(2)}</span>
        <span>{preferredCurrency}</span>
      </p>
}
```

Reply:
Refactored.